cmd/helpers.go

```diff
@@ -2,17 +2,18 @@ package cmd
 
 import (
 	"fmt"
+	"runtime"
 
 	"github.com/jetstack/preflight/pkg/client"
 	"github.com/jetstack/preflight/pkg/version"
 )
 
 func printVersion(verbose bool) {
-	fmt.Println("Preflight version: ", version.PreflightVersion, version.Platform)
+	fmt.Println("Preflight version: ", version.PreflightVersion, runtime.GOOS+"/"+runtime.GOARCH)
```

Comment:
**Self-review:** I've (somewhat) checked that the old `version.Platform` had the same values as the new `runtime.GOOS+"/"+runtime.GOARCH` since it used to be set to

```
-X "github.com/jetstack/preflight/pkg/version.Platform=$(GOOS)/$(GOARCH)" \
```

which were set using 

```
GOOS:=$(shell go env GOOS)
GOARCH:=$(shell go env GOARCH)
```

which matches the new Go string above.